addons/stock/models/stock_quant.py

```diff
@@ -863,7 +863,7 @@ def _get_quants_action(self, domain=None, extend=False):
             action['id'] = target_action.id
 
         if self.env.context.get('inventory_mode') and self.user_has_groups('stock.group_stock_manager'):
-            action['view_id'] = self.env.ref('stock.view_stock_quant_tree_editable').id
+            action['view_id'] = self.env.ref('stock.view_stock_quant_tree_inventory_editable').id
```

Comment:
A few minor concerns:
1. I think having some of the inventory adjustment related buttons in the Update Quantity view a bit confusing. For example, "Set"/"Clear"/"Apply":
 - if you click "Set" the "Counted Quantity" becomes 0 instead of "Available Quantity"
 - if you click "Set" and then "Clear", it sets the "Available Quantity" to 0 (instead of leaving it as is)
 - "Apply" is mainly just confusing since the Counted Quantity is auto-applied as soon as you type it in.
 - The multi-record buttons have similar issues (e.g. "Set" does correctly work, but using "Apply afterwards will wipe the qtys again)

2. Making the same view for "Update Quantity" means that we can see all the buttons (e.g. "History", "Apply", "Clear") on lines for the "Virtual Locations/Inventory adjustment" (these are excluded from the inventory adjustment). We can also have this line selected when doing multi-record actions (e.g. "Request A Count", "Set"). This is mainly fine because most of the actions don't do anything to the line, except if the buttons are pushed for another line in the view => "Apply" can opening the "Conflict in Inventory Adjustment" wizard and get stuck in it. It's unlikely this will happen often, but just in case to reproduce:
 - Create a new product + Update Quantity (to any non-0 value)
 - Reload view with no filters applied (Inventory adjustment "Difference" = whatever non-0 value you put in)
 - "Set" + "Apply" your quant internal location line
 - "Apply" the "Inventory adjustment" line

3. Easiest comment :smiley: : The "Update Quantity" had an extra column not in the inventory adjustment view: `removal_date`, do we want to keep it?